wordpress-mobile/WordPress-iOS
# Site Tags search: add 'no tags matching your search' view.

Fixes #10143 

To test:
- On a site with tags, go to site > settings > Tags.
- Enter an invalid search term.
- Verify the label appears, and is static at the top of the view.

**NOTE**: There is a pre-existing issue with the search bar not displaying correctly on the iPad (https://github.com/wordpress-mobile/WordPress-iOS/issues/10145).

iPhone portrait:
![iphone_port](https://user-images.githubusercontent.com/1816888/45520667-a9a5b280-b777-11e8-81d7-f092d0a3607e.png)

iPhone landscape:
![iphone_land](https://user-images.githubusercontent.com/1816888/45520672-af02fd00-b777-11e8-8474-1b266150becc.png)

iPad portrait:
![ipad_port](https://user-images.githubusercontent.com/1816888/45520576-3308b500-b777-11e8-9d3c-688d7bd9534f.png)

iPad landscape:
![ipad_land](https://user-images.githubusercontent.com/1816888/45520592-4287fe00-b777-11e8-9336-70688b35f80f.png)
